src/main/java/frc/robot/constants/RobotConstants.java

```diff
@@ -14,10 +14,12 @@ public static final class NeoMotorConstants {
   public static final class DriveConstants {
     // placeholder CAN IDs, fix these later
 
-    public static final double kFrontLeftChassisAngularOffset = 0.0;
-    public static final double kFrontRightChassisAngularOffset = 0.0;
-    public static final double kBackLeftChassisAngularOffset = 0.0;
-    public static final double kBackRightChassisAngularOffset = 0.0;
+    public static final double kMoveInputRotationalOffset = -Math.PI / 2;
```

Comment:
Remove this through a cherrypick of a commit from the revived drivetrain.

Reply:
Which commit?

Reply:
Probably 5c64787

Reply:
I can't find a good commit for this. Would it be fine to make changes directly on this branch and resolve minor merge conflicts when we rebase develop into drivetrain or PR drivetrain into develop?

Reply:
No, wait for Joseph to recreate drivetrain. He plans on recreating it by rebranching off of develop and one of the commits in the process will delete this line.

Reply:
Due to lack of time, I'm going to resolve this conversation with appropriate changes done directly on this branch.